src/command_helpers.rs

```diff
@@ -414,28 +387,18 @@ pub(crate) fn spawn(
         Ok(child) => Ok(child),
         Err(ref e) if e.kind() == io::ErrorKind::NotFound => {
             let extra = if cfg!(windows) {
-                " (see https://docs.rs/cc/latest/cc/#compile-time-requirements \
-for help)"
+                " (see https://docs.rs/cc/latest/cc/#compile-time-requirements for help)"
             } else {
                 ""
             };
             Err(Error::new(
                 ErrorKind::ToolNotFound,
-                format!(
-                    "Failed to find tool. Is `{}` installed?{}",
-                    program.display(),
-                    extra
-                ),
+                format!("failed to find tool {:?}{extra}: {e}", cmd.0.get_program()),
```

Comment:
I think we should display error before the extra?

```suggestion
                format!("failed to find tool {:?}: {e}{extra}", cmd.0.get_program()),
```

Reply:
I was unsure, but let's do it your way